packages/elements/src/components/API/APIWithResponsiveSidebarLayout.tsx

```diff
@@ -75,17 +83,20 @@ export const APIWithResponsiveSidebarLayout: React.FC<SidebarLayoutProps> = ({
       name={serviceNode.name}
     >
       {node && (
-        <ParsedDocs
-          key={pathname}
-          uri={pathname}
-          node={node}
-          nodeTitle={node.name}
-          layoutOptions={layoutOptions}
-          location={location}
-          exportProps={exportProps}
-          tryItCredentialsPolicy={tryItCredentialsPolicy}
-          tryItCorsProxy={tryItCorsProxy}
-        />
+        <ElementsOptionsProvider renderExtensionAddon={renderExtensionAddon}>
```

Comment:
For some reason the `API*` components don't see to wrap around the `ElementsOptionsProvider`. This is required to be able to use `useOptionsCtx` in `elements-core`. 

I appreciate some guidance if this the right approach you had in mind.